packages/react-core/src/components/JumpLinks/JumpLinksItem.tsx

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/JumpLinks/jump-links';
+import buttonStyles from '@patternfly/react-styles/css/components/Button/button';
 import { JumpLinksList } from './JumpLinksList';
 
 export interface JumpLinksItemProps extends Omit<React.HTMLProps<HTMLLIElement>, 'onClick'> {
   /** Whether this item is active. Parent JumpLinks component sets this when passed a `scrollableSelector`. */
   isActive?: boolean;
   /** Href for this link */
-  href?: string;
+  href: string;
```

Comment:
This needs to be required, as without it the anchor internal to the component cannot receive keyboard focus.

If the intent is that a consumer can pass either an href to link to an ID on the page, or the onClick to handle this functionality themselves, then we would need to render either an anchor with an href or a button with the onClick, not an anchor with either/both.

Reply:
Should we make a codemod to at least warn consumers about this?

Reply:
https://github.com/patternfly/pf-codemods/issues/559